pkgs/ffigen/lib/src/code_generator/objc_built_in_functions.dart

```diff
@@ -71,13 +71,36 @@ class ObjCBuiltInFunctions {
     'NSFastEnumerationState',
     'NSRange',
   };
+  static const builtInEnums = {
```

Comment:
For all these built-in lists. Is there a way to generate them (possibly with FFIgen itself)?

Benefits:

* We won't miss any of existing ones
* If new ones are added we just rerun the script.